pages/common/rc.md

```diff
@@ -0,0 +1,12 @@
+# rc
+
+> Rustcat is a modern port listener & Reverse Shell that is very easy to use.
```

Comment:
Let's mention it is similar to `nc`
```suggestion
> Rustcat is a modern port listener & Reverse Shell that is very easy to use.
> Similar to `nc`.
```

Reply:
I have edited the comment from @mfrw to change 1 line.